fdbclient/ClientLogEvents.h

```diff
@@ -108,6 +108,41 @@ namespace FdbClientLogEvents {
 		}
 	};
 
+	// Version V3 of EventGetVersion starting at 6.3
+	struct EventGetVersion_V3 : public Event {
+		EventGetVersion_V3(double ts, double lat, uint32_t type, Version version) : Event(GET_VERSION_LATENCY, ts), latency(lat), readVersion(version) {
+			if(type == GetReadVersionRequest::PRIORITY_DEFAULT) {
+				priorityType = PRIORITY_DEFAULT;
+			} else if (type == GetReadVersionRequest::PRIORITY_BATCH) {
+				priorityType = PRIORITY_BATCH;
+			} else if (type == GetReadVersionRequest::PRIORITY_SYSTEM_IMMEDIATE){
+				priorityType = PRIORITY_IMMEDIATE;
+			} else {
+				ASSERT(0);
+			}
+		 }
+		EventGetVersion_V3() { }
+
+		template <typename Ar>	Ar& serialize(Ar &ar) {
+			if (!ar.isDeserializing)
+				return serializer(Event::serialize(ar), latency, priorityType, readVersion);
+			else
+				return serializer(ar, latency, priorityType, readVersion);
+		}
+
+		double latency;
+		TrasactionPriorityType priorityType {PRIORITY_END};
+		Version readVersion;
+
+		void logEvent(std::string id, int maxFieldLength) const {
+			TraceEvent("TransactionTrace_GetVersion")
+			.detail("TransactionID", id)
+			.detail("Latency", latency)
+			.detail("PriorityType", priorityType)
+			.detail("ReadVersion", readVersion);
+		}
+	};
+
```

Comment:
This works, but it feels a bit awkward to copy the entire struct to just add a new field. We should consider an alternative approach that allows more surgical changes.